src/classes/FORM_ServiceGiftEntry.cls

```diff
@@ -423,5 +423,36 @@ public with sharing class FORM_ServiceGiftEntry {
         }
     }
 
+    /*******************************************************************************************************
+    * @description @description Creates a 'unique' id made to look like a UUID..
+    *
+    * @return {String} guid: Generated UUID.
+    */
+    private static String generateId() {
```

Comment:
I think we've discussed as a team that this class could maybe have a better name, since many of the methods here (like this one, and the populateColumnHeaders() method that calls it) are supporting the Gift Entry app more than anything form-specific. But I know we are staggering the delivery of all this refactor so not a big deal if we tackle that in a future WI!

Reply:
This is a good point, and illuminates the fact that Gift Entry classes should probably be unified under the GE prefix with the FORM prefix being removed entirely.  I created a [draft PR here](https://github.com/SalesforceFoundation/NPSP/pull/5863) to view what that would look like.  I'll reach out to try to discuss today if we want to incorporate these changes into this branch/PR and bring this work to its logical conclusion within this WI.